### PR TITLE
Add DyND to organization/team.md

### PR DESCRIPTION
**Pull request by mwiebe**

### CODE REVIEW

**Comment by moorepants** [comment]
Please add your org to the main README too.

**Comment by mwiebe** [comment]
I've added to the readme except for a patch, doing that now.

File: README.md [hunk 1]
```diff
@@ -25,6 +25,21 @@ In alphabetic order.
 
   <tr>
     <td>
+      <img width="300px" src=""/>
```
Comment (moorepants) [inline README.md]: Just delete this image tag if you don't have one.

**Comment by moorepants** [comment]
What do you mean by "except for a patch"?

**Comment by mwiebe** [comment]
The image. Sorry, I looked at the word "patch" and visualized some kind of logo patch, exactly as these images are...apparently I'm just making things difficult for myself. :)

**Comment by mwiebe** [comment]
Fixed, should be good to go now.

**Pull request merged**